assets/locales/vi.json

```
@@ -160,6 +164,13 @@
     "dkt_intro": "DKT được thành lập với niềm đam mê và khát vọng thành công trong lĩnh vực Thương mại điện tử. Chúng tôi đã và đang khẳng định được vị trí hàng đầu với những sản phẩm."
   },
   "product": {
-    "title": "Danh sách sản phẩm"
+    "title": "Danh sách sản phẩm",
+    "blog_tags": "Blog tags"},
```

```suggestion
    "title": "Danh sách sản phẩm",
    "blog_tags": "Blog tags"
  },
```
Em bổ sung text blog tags này bên files tiếng anh nữa nha